doc/rst/source/explain_-R.rst_

```diff
@@ -39,9 +39,9 @@ The **-R** option defines the map region or data domain of interest. It may be s
 #. **-R**\ *code1,code2,...*\ [**+e**\ \|\ **r**\ \|\ **R**\ *incs*]. This indirectly supplies the region by
    consulting the DCW (Digital Chart of the World) database and derives the bounding regions for one or more
    countries given by the codes. Simply append one or more comma-separated countries using either the two-character
-   `ISO 3166-1 alpha-2 convention <https://en.wikipedia.org/wiki/ISO_3166-1_alpha-2>`_ (e.g., NO) or the full
-   country name (e.g., Norway). To select a state within a country (if available), append .state (e.g, US.TX),
-   or the full state name (e.g., Texas). To specify a whole continent, spell out the full continent name (e.g., -RAfrica).
+   `ISO 3166-1 alpha-2 convention <https://en.wikipedia.org/wiki/ISO_3166-1_alpha-2>`_ (e.g., -RNO) or the full
+   country name (e.g., -RNorway). To select a state within a country (if available), append .state (e.g, -RUS.TX),
+   or the full state name (e.g., -RTexas). To specify a whole continent, spell out the full continent name (e.g., -RAfrica).
```

Comment:
Please remove the ", or the full state name (e.g., -RTexas)".

The rest is good. The subject of this paragraph is -R scope so no harm to the -E option that is not called for business here.

Reply:
Please be my guest to edit.
1. This is already getting over my head. Time to hand this over to experts.
2. I mess up re-edits.